vars/neoDeploy.groovy

```diff
@@ -87,6 +87,21 @@ def call(parameters = [:]) {
                                                       stepConfiguration, stepConfigurationKeys,
                                                       ConfigurationLoader.defaultStepConfiguration(script, stepName))
 
+        HOME_CHECK: {
```

Comment:
* The behaviour with validation inside the ui5 reference pipeline was:
  * NEO_HOME needs to be provided from the config. If it was not provided, we failed. The code with launching neo.sh from the path was not used.
* The behaviour now is:
  * In case NEO_HOME is provided (either from config or as Jenkins environment variable), we validate. In case it is not provided we try to launch neo.sh from the path. This is the behaviour we had already when using the step outside our ui5 pipeline.

Reply:
But if NEO_HOME is in the path the neo version must be validated anyways.

Reply:
Agreed, but currently this is not possible with the current version of toolValidate. --> Should be done in another PR.
Some more background: using ```which``` we get the full path to the executable. toolValidate works upon ```NEO_HOME``` which is not the path to the executable. In case ```which neo.sh``` returns a symbolic link (typically provided in ```/usr/bin```or ```/usr/local/bin```) it is not even possible to strip the trailing ```tools/neo.sh```.